collect_app/src/main/java/org/odk/collect/android/utilities/HtmlUtils.java

```diff
@@ -77,32 +77,40 @@ static String markdownToHtml(String text) {
                 text, createSpan);
 
         //intermediary replacements keys for special characters, N/B: These symbols are not meant to be interpreted as markdown
-        text = text.replaceAll("(?s)\\\\#", "&#35;");
```

Comment:
`(?s)` in regex is calleda  dot-all mode.
In the default mode (without (?s)), the dot (.) character in a regex pattern matches any character except a newline (\n). However, when (?s) is used, the dot (.) matches any character, including newline characters.
so it's about using dots but the lines I updated by removing `(?s)` didn't contain any.